contrib/puppet/anycast_healthchecker/templates/check.conf.erb

```diff
@@ -1,4 +1,5 @@
 [<%= @name%>]
+interface = <%= @name %>
```

Comment:
In order for this to work you need to add `interface` as parameter to the `anycast_healthchecker::check` defined type class and set a `lo` as default value. Then change `<%= @name %>` to `<%= @interface %>` in the template.

Reply:
Not 'have to', as name can be the interface (which is what I've had going on here).  But, point taken.